### PR TITLE
Allow leaderboard color customisation

### PR DESCRIPTION
**Pull request by ClementPasteau**
Allow customizing 4 colors for the leaderboard of a game, to improve the embedded experience of a leaderboard in-game.
For subscribed users only.

https://gdevelop-storybook.s3.amazonaws.com/allow-leaderboard-color-customisation/latest/index.html?path=/story/leaderboard-leaderboardappearancedialog--without-subscription

image
image

### CODE REVIEW

File: newIDE/app/src/fixtures/GDevelopServicesTestData/index.js [hunk 1]
```diff
@@ -301,7 +333,7 @@ export const fakeNoSubscriptionAuthenticatedUser: AuthenticatedUser = {
   firebaseUser: indieFirebaseUser,
   subscription: noSubscription,
   usages: usagesForIndieUser,
-  limits: limitsForIndieUser,
+  limits: limitsForNoSubscriptionUser,
```
Comment (4ian) [inline newIDE/app/src/fixtures/GDevelopServicesTestData/index.js:336]: 👍

File: newIDE/app/src/GameDashboard/LeaderboardAdmin/GetSubscriptionCard.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    borderRadius: 10,
+    alignItems: 'center',
+  },
+  diamondIcon: {
+    width: 50,
+    height: 50,
+  },
+};
+
+const GetSubscriptionCard = () => {
```
Comment (4ian) [inline newIDE/app/src/GameDashboard/LeaderboardAdmin/GetSubscriptionCard.js:24]: Let's remember to factor this in the UI folder next time we use it.

File: newIDE/app/src/UI/ColorField/ColorPicker.js [hunk 3]
```diff
@@ -49,67 +50,73 @@ const styles = {
   },
 };
 
-class ColorPicker extends React.Component<Props, State> {
-  _swatch = React.createRef<HTMLDivElement>();
-  state = {
-    displayColorPicker: false,
-  };
+const ColorPicker = ({
```
Comment (4ian) [inline newIDE/app/src/UI/ColorField/ColorPicker.js:53]: Nice!

**Review by 4ian — APPROVED** [review]
lgtm, I like it!

**Comment by AlexandreSi** [comment]
Could the disabled color inputs display `Background color` only and not `RGB like 100;200;80`, I find it confusing

**Comment by AlexandreSi** [comment]
I don't know if the highlight colors should be in the Score section. I had troubles figuring out what Score highlight is.

**Review by AlexandreSi — APPROVED** [review]
LGTM

**Pull request merged**